src/main/java/TaskList.java

```diff
@@ -0,0 +1,263 @@
+import java.time.format.DateTimeParseException;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Objects;
+
+public class TaskList {
+    List<Task> taskList;
+    Storage storage;
+
+    public TaskList(Storage storage) {
+        this.storage = storage;
+        this.taskList = storage.loadTask();
+    }
+
+    /**
+     * Adds task of taskName to taskList.
+     * @param taskName Task to be added to taskList
+     */
+    public void addTask(String taskName) {
+        try {
+            String taskType = taskName.split(" ", 2)[0];
+            String taskDesc = taskName.split(" ", 2)[1];
```

Comment:
Always use the parser class for parsing of Strings and commands which are based on input by the user. It helps contain logically related code to one file and makes it easy to understand the code.